lib/migration-stream.js

```diff
@@ -35,16 +35,19 @@ function MigrationStream() {
 
     if (v2 === true) {
       decompressBuffer(tile.buffer, function(err, decompressedBuffer){
-          if (err) return callback(err); 
+          if (err) {
+            err.code = 'EINVALID';
+            return callback(err);
+          } 
 
           vtvalidate.isValid(decompressedBuffer, function(err, result){
             // returns empty string if it's a valid tile
             // if invalid tile - string that specifies why the tile is invalid
             if (result === 'ClosePath command count is not 1' || result === 'Max count too large'){
-              
-              var error = new Error(`Fatal Error: ${result}`); 
-              return callback(error); 
-
+              var error = new Error(`Fatal Error: ${result}`);
```

Comment:
We do not need the `Fatal Error:` string here. It can just be `var error = new Error(result);`